packages/mui-material/package.json

```diff
@@ -104,5 +104,18 @@
   },
   "engines": {
     "node": ">=12.0.0"
+  },
+  "pigment-css": {
+    "vite": {
+      "include": [
+        "prop-types",
+        "react-is",
+        "hoist-non-react-statics",
+        "react",
+        "react-dom",
+        "@emotion/react",
+        "@emotion/styled"
```

Comment:
```suggestion
```
I got it working without `@emotion/styled`

Reply:
@siriwatknp I was able to make it work without `@emotion/react` as well. Can you confirm if this is the same for you ?